sir_cake/seller/views.py

```diff
@@ -79,3 +81,40 @@ def delete_item(request, pk):
         messages.success(request, "Item successfully deleted")
         return redirect('all_items')
     return render(request, 'seller/delete-item.html', {'item': item})
+
+
+def orders(request):
+    orders = Order.objects.exclude(
+        status='preorder').order_by('-order_date')
+
+    order_status = request.GET.get('status')
+    if order_status is None:
+        order_status = 'ALL'
```

Comment:
It would be nice to have order status values as constants in the `Order` model. You would then use them as `Order.Status.ALL` etc. You would then use these constants everywhere instead of using the string value (e.g. `"ALL"`) directly.